### PR TITLE
Feature/#154 project detail application status

### PR DESCRIPTION
**Pull request by kenzo23312**
Status of uploaded concept note

### CODE REVIEW

File: frontend/src/reducers/conceptNote.js [hunk 1]
```diff
@@ -40,12 +48,33 @@ const clearConceptNoteState = (state) => {
   return R.assoc('cnFile', false, clearLocalFile);
 };
 
+const saveConceptNote = (state, action) => {
+  const file = R.assoc('cnFile', action.response.cn, state);
+  return R.assoc('created', action.response.created, file);
+};
+
+export const projectApplicationExists = partnerId => (dispatch) => {
```
Comment (marcindo) [inline frontend/src/reducers/conceptNote.js]: this function is also duplicated

File: frontend/src/reducers/projectApplication.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      dispatch(projectAppLoadEnded());
+      dispatch(projectAppLoadSuccess(profiles));
+    })
+    .catch((error) => {
+      dispatch(projectAppLoadEnded());
+      dispatch(projectAppLoadFailure(error));
+    });
+};
+
+export default function countryProfilesReducer(state = initialState, action) {
```
Comment (marcindo) [inline frontend/src/reducers/projectApplication.js]: proper name

File: frontend/src/components/eois/details/cfeiDetailsHeader.js [hunk 3]
```diff
@@ -52,10 +59,19 @@ class CfeiHeader extends Component {
     history.push(`/cfei/${type}`);
   }
 
+  cfeiTabs() {
+    return this.props.tabs.map((tab, index) => {
+      if (index === 1) {
+        return <CustomTab label={tab.label} key={index} checked={this.props.cnFile} />;
```
Comment (marcindo) [inline frontend/src/components/eois/details/cfeiDetailsHeader.js]: you use cnFile from different reducer than the one which checks endpoint for partner application

File: frontend/src/reducers/projectApplication.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+export const projectAppLoadEnded = () => ({ type: PROJECT_APP_LOAD_ENDED });
+
+const saveApplication = (state, action) => R.assoc('app', action.response, state);
+
+const initialState = {
+  loading: false,
+  app: null,
+};
+
+export const projectApplicationExists = partnerId => (dispatch) => {
```
[the remark this answers is not in the document]
Comment (kenzo23312) [reply frontend/src/reducers/projectApplication.js]: done

File: frontend/src/reducers/conceptNote.js (hunk 1 above)
Comment (kenzo23312) [reply frontend/src/reducers/conceptNote.js]: done

File: frontend/src/reducers/conceptNote.js [hunk 5]
```diff
@@ -17,6 +17,16 @@ export const UPLOAD_CN_CLEAR_ERROR = 'UPLOAD_CN_CLEAR_ERROR';
 export const UPLOAD_CN_CLEAR = 'UPLOAD_CN_CLEAR';
 export const UPLOAD_CN_LOCAL_FILE = 'UPLOAD_CN_LOCAL_FILE';
 
+export const PROJECT_APP_LOAD_STARTED = 'PROJECT_APP_LOAD_STARTED';
```
[the remark this answers is not in the document]
Comment (kenzo23312) [reply frontend/src/reducers/conceptNote.js]: done

**Pull request merged**